org.eclipse.scanning.example.xcen.ui/src/org/eclipse/scanning/example/xcen/ui/XcenServices.java

```diff
@@ -35,26 +32,29 @@ public void setLoaderService(ILoaderService loaderService) {
 
 	private static XcenServices current;
 
-	public void start(ComponentContext context) {
-		current = this;
+	public void  start(ComponentContext context) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "context". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)